app/controllers/admin/creator_settings_controller.rb

```diff
@@ -34,5 +38,11 @@ def extra_authorization
     def settings_params
       params.permit(ALLOWED_PARAMS)
     end
+
+    def upload_logo(image)
+      ArticleImageUploader.new.tap do |uploader|
```

Comment:
I'm using what I found in the image uploader.

Reply:
We probably want to change the logo to have it's own Uploader with its own set of configuration rules (liek resizing etc.) rather than re-using the Article Image Uploader, but we can change that once we have things hooked up.